shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/visitor/MySQLDCLVisitor.java

```diff
@@ -82,4 +89,21 @@ public ASTNode visitRenameUser(final RenameUserContext ctx) {
     public ASTNode visitSetPassword(final SetPasswordContext ctx) {
         return new SetPasswordStatement();
     }
+
+    @Override
+    public ASTNode visitGrant(final GrantContext ctx) {
+        GrantStatement result = new GrantStatement();
+        PrivilegeClause_Context privilegeClause = ctx.privilegeClause_();
+        if (null != privilegeClause && null != privilegeClause.onObjectClause_()) {
+            PrivilegeLevel_Context privilegeLevel = privilegeClause.onObjectClause_().privilegeLevel_();
```

Comment:
`PrivilegeLevel_Context` is visited, so please rename `PrivilegeLevel_` to `PrivilegeLevel`. Notice: replace this key word all of our project.

Reply:
done